test/e2e/mxGraph.view.test.ts

```diff
@@ -46,7 +46,7 @@ async function expectTask(cellId: string, expectedText: string): Promise<void> {
 
 async function expectSequenceFlow(cellId: string, expectedText?: string): Promise<void> {
   const svgElementHandle = await page.waitForSelector(`#${graphContainerId} svg g g[data-cell-id="${cellId}"]`);
-  expect(await svgElementHandle.evaluate(node => node.getAttribute('class'))).toBe('class-state-cell-style-normal');
+  expect(await svgElementHandle.evaluate(node => node.getAttribute('class'))).toBe('class-state-cell-style-sequenceFlow;normal');
```

Comment:
:question: shouldn't we have `-` instead of `;` for consistency with the rest of the class name?
I am asking this if we want this to be something use for extension/interaction of our lib in the future

Reply:
I forgot that we calculated it. I tought that it was MxGraph.
I changed it 😉